scripts/start_argilla_server.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/bash
+set -e
+
+# Check elasticsearch response status
+status_code=$(curl -s -o /dev/null -w "%{http_code}" "$ARGILLA_ELASTICSEARCH")
```

Comment:
Do we still need this step? The `wait-for-it.sh` script should finish once the elasticsearch is up and running, right? So, the start script will be run once the elasticsearch is ready.

Reply:
I think this is an extra check if the elasticsearch returning the expected output or not